### ISSUE TITLE
Multisig: clarify why an Accept or Reject button is disabled

### ISSUE THREAD

**Issue by Swader** [polkadot-js/apps#7588]
We are encountering problems on the Statemine side of things with the Multisig UI. 

Specifically, the accept and reject buttons are always disabled without an indication why, despite the call being submitted successfully and the deposit being locked and the hash being selectable in the UI. 

Submitting manually via extrinsics produces the Signatories our of order error on approvals, which we are not sure how to debug given that we follow the precise order in which the multisig was created.

Would it be possible to clarify the issue behind the disabled button some more?

**Comment by jacogr** [comment]
It will be disabled when an invalid queryInfo result for the weight param is retrieved or none retrieved (yet). Since Statemine does have payment queryInfo, could you check to see if you have any errors in the console?

Will also just run through tests (again).

**Comment by Swader** [comment]
No errors in the console - not sure when they should fire, on UI load there are none and when I open the multisig popup nothing happens in terms of calls that would populate the console.

This multisig call is for moving RMRK assets and was composed through the UI and so no weight was input by us, but I have also composed another through the extrinsics page and got the same result (in that case I used some absurdly high weight to be safe). I now have 3 pending hashes but am unable to cancel or accept them.
